<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWnd</class>
 <widget class="QMainWindow" name="MainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ANNA</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>48</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>65536</width>
        <height>48</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Model file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ModelPath"/>
       </item>
       <item>
        <widget class="QPushButton" name="ModelFindButton">
         <property name="maximumSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>AI</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="AINameBox">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>65536</width>
              <height>27</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>Anna:</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&lt;none&gt;</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="ChatLog">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="AttachmentsList">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>65536</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>User</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QComboBox" name="UserNameBox">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>65536</width>
              <height>27</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>USER:</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>User:</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="UserInput"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="UserInputOptions">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>144</width>
            <height>65536</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="SendButton">
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="NewlineCheck">
             <property name="text">
              <string>no newline</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="SamplingCheck">
             <property name="text">
              <string>no sampling</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="AttachButton">
             <property name="text">
              <string>Attach...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="BeforeRadio">
             <property name="text">
              <string>Before</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="AfterRadio">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>After</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave_dialog">
     <property name="title">
      <string>Save dialog</string>
     </property>
     <addaction name="actionPlain_text"/>
     <addaction name="actionMarkdown"/>
     <addaction name="actionHTML"/>
    </widget>
    <addaction name="actionNew_dialog"/>
    <addaction name="menuSave_dialog"/>
    <addaction name="separator"/>
    <addaction name="actionSave_state"/>
    <addaction name="actionLoad_state"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSimple_view"/>
    <addaction name="actionAdvanced_view"/>
    <addaction name="actionProfessional_view"/>
    <addaction name="separator"/>
    <addaction name="actionClear_attachments"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionOnline_help"/>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>Model</string>
    </property>
    <addaction name="actionLoad_model"/>
    <addaction name="actionLoad_vision_encoder"/>
    <addaction name="actionLoad_initial_prompt"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuModel"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_dialog">
   <property name="text">
    <string>New dialog</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave_state">
   <property name="text">
    <string>Save state</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionLoad_state">
   <property name="text">
    <string>Load state</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOnline_help">
   <property name="text">
    <string>Online help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSimple_view">
   <property name="text">
    <string>Simple view</string>
   </property>
  </action>
  <action name="actionAdvanced_view">
   <property name="text">
    <string>Advanced view</string>
   </property>
  </action>
  <action name="actionProfessional_view">
   <property name="text">
    <string>Professional view</string>
   </property>
  </action>
  <action name="actionLoad_model">
   <property name="text">
    <string>Load model</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionLoad_vision_encoder">
   <property name="text">
    <string>Load vision encoder</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="actionLoad_initial_prompt">
   <property name="text">
    <string>Load initial prompt</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionPlain_text">
   <property name="text">
    <string>Plain text</string>
   </property>
  </action>
  <action name="actionMarkdown">
   <property name="text">
    <string>Markdown</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionClear_attachments">
   <property name="text">
    <string>Clear attachments</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
